reporter-web-app/src/models/WebAppPackage.js

```diff
@@ -621,6 +631,10 @@ class WebAppPackage {
             && this.#concludedLicense.length !== 0;
     }
 
+    hasAuthors() {
```

Comment:
Functions are alphabetically sorted so hasAuthors() should be in front of hasConcludedLicense()